x-pack/plugins/enterprise_search/public/applications/app_search/routes.ts

```diff
@@ -41,7 +41,7 @@ export const ENGINE_CRAWLER_PATH = `${ENGINE_PATH}/crawler`;
 
 export const META_ENGINE_SOURCE_ENGINES_PATH = `${ENGINE_PATH}/engines`;
 
-export const ENGINE_RELEVANCE_TUNING_PATH = `${ENGINE_PATH}/search-settings`;
+export const ENGINE_RELEVANCE_TUNING_PATH = `${ENGINE_PATH}/relevance-tuning`;
```

Comment:
The the view and all view components, I am using Relevance Tuning over search settings.

Reply:
+10000000000000000

Reply:
Quick heads up (I'll continue reminding folks as we port over pages, no worries), URLs should be snake_cased in Kibana, not kebab-cased - I have them kebab-case right now so that they still work when generating URLs to App Search.

```suggestion
export const ENGINE_RELEVANCE_TUNING_PATH = `${ENGINE_PATH}/relevance_tuning`;
```

Reply:
755e26beab0c650491c978f41ba368fd6872ca2b